changeset/funcs.go

```diff
@@ -6,16 +6,6 @@ import (
 	"github.com/canopyclimate/golive/htmltmpl"
 )
 
-// Funcs returns a map of functions that can be used in templates:
-//   - inputTag: renders an input tag with the given key and value for the provided changeset
-//   - errorTag: renders an error tag if there is an error for the given key in the provided changeset
-func Funcs() htmltmpl.FuncMap {
```

Comment:
Why did these go away?

Reply:
Only logical thing we could type these functions with is `any` but that causes a runtime error unless the `Changeset` type is `any` which is obviously not helpful.

Reply:
See my suggestion below...